internal/format/testdata/tfvars/hcl-PrintDescription.golden

```diff
@@ -0,0 +1,104 @@
+
+# A variable with underscores.
+input_with_underscores = ""
+
+# It's list number two.
+list-2                 = ""
+
+# It's map number two.
+map-2                  = ""
+
+# It's number number two.
+number-2               = ""
+
+# It's string number two.
+string-2               = ""
+
+string_no_default      = ""
+unquoted               = ""
```

Comment:
Description comments are completely fine here, but the whole file is not terraform formatted properly. It should like like something like this:

```hcl
# A variable with underscores.
input_with_underscores = ""

# It's list number two.
list-2 = ""

# It's map number two.
map-2 = ""

# It's number number two.
number-2 = ""

# It's string number two.
string-2 = ""

string_no_default = ""
unquoted          = ""

...
```

Reply:
@khos2ow Thank you for your comment and this formatted example.
Now I fix the code to generate terraform-formatted tfvars.